src/main/java/com/codessquad/qna/service/AnswerService.java

```diff
@@ -0,0 +1,52 @@
+package com.codessquad.qna.service;
+
+import com.codessquad.qna.domain.Answer;
+import com.codessquad.qna.domain.Question;
+import com.codessquad.qna.domain.User;
+import com.codessquad.qna.exception.NotFoundException;
+import com.codessquad.qna.repository.AnswerRepository;
+import com.codessquad.qna.repository.QuestionRepository;
+import com.codessquad.qna.util.HttpSessionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpSession;
+import java.util.List;
+
+@Service
+public class AnswerService {
+    Logger logger = LoggerFactory.getLogger(AnswerService.class);
+    private final AnswerRepository answerRepository;
+    private final QuestionRepository questionRepository;
+
+    @Autowired
+    public AnswerService(AnswerRepository answerRepository, QuestionRepository questionRepository) {
+        this.answerRepository = answerRepository;
+        this.questionRepository = questionRepository;
+    }
+
+    public Answer getOneById(long answerId) {
+        return answerRepository.findByAnswerIdAndDeletedFalse(answerId)
+                .orElseThrow(() -> new NotFoundException("존재하지 않는 답변입니다."));
+    }
+
+    public void create(Long id, String contents, HttpSession session) {
+        User loginUser = HttpSessionUtils.getUserFromSession(session);
+        Question question = questionRepository.findByQuestionIdAndDeletedFalse(id)
+                .orElseThrow(() -> new NotFoundException("존재하지 않는 질문입니다."));
+
+        answerRepository.save(new Answer(question, contents, loginUser));
+    }
+
+    public void remove(Answer answer) {
+        answer.deleted();
+        Answer testAnswer = answerRepository.save(answer);
+        logger.info("testAnswer: " + testAnswer.toString());
```

Comment:
로거는 스트링 템플릿을 지원합니다. 이 편이 [성능에도 이점이 있습니다.](https://stackoverflow.com/questions/10555409/logger-slf4j-advantages-of-formatting-with-instead-of-string-concatenation) 
앞으로는 템플릿을 적극적으로 사용해 주시면 좋을 것 같습니다. 

```suggestion
        logger.info("testAnswer: {}", testAnswer);

Reply:
그렇군요! 앞으로 템플릿을 잘 확인하고 사용해보겠습니다 :)